src/compiler/factory/emitHelpers.ts

```diff
@@ -886,6 +888,7 @@ namespace ts {
             templateObjectHelper,
             spreadHelper,

```

Comment:
@sheetalkamat, do I need to leave these in for project references leveraging `.tsbuildinfo`, or can I safely remove them?

Reply:
you dont need to leave these since new version = new tsbuildinfo.